bundles/org.openhab.binding.opengarage/src/main/java/org/openhab/binding/opengarage/internal/OpenGarageHandler.java

```diff
@@ -0,0 +1,165 @@
+                    changeStatus(((OnOffType) command).equals(OnOffType.ON));
+                    return;
+                }
+                break;
+        }
+
+        if (RefreshType.REFRESH != command) {
+            logger.warn("Received command {} of wrong type for thing '{}' on channel {}", command, thing.getUID().getAsString(),
+                    channelUID.getId());
+        } 
+    }
+
+    @Override
+    public void initialize() {
+        logger.debug("Initializing Opengarage Unit");
+        OpenGarageConfiguration config = getConfigAs(OpenGarageConfiguration.class);
+        logger.debug("config.host = {}, refresh = {}, password = {}, port = {}", config.host, config.refresh, config.password, config.port);
+        if (config.host == null) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, "Host address must be set");
+        } else {
+            webTargets = new OpenGarageWebTargets(config.host, config.port, config.password);
+            refreshInterval = config.refresh;
+
+            schedulePoll();
+        }
+    }
+
+    @Override
+    public void handleRemoval() {
+        super.handleRemoval();
+        stopPoll();
+    }
+
+    @Override
+    public void dispose() {
+        super.dispose();
+        stopPoll();
+    }
+
+    private void schedulePoll() {
+        if (pollFuture != null) {
+            pollFuture.cancel(false);
+        }
+        logger.debug("Scheduling poll for 500ms out, then every {} s", refreshInterval);
+        pollFuture = scheduler.scheduleWithFixedDelay(this::poll, 1, refreshInterval, TimeUnit.SECONDS);
+    }
+
+    private synchronized void stopPoll() {
+        if (pollFuture != null && !pollFuture.isCancelled()) {
+            pollFuture.cancel(true);
+            pollFuture = null;
+        }
+    }
+
+    private synchronized void poll() {
+        try {
+            logger.debug("Polling for state");
+            pollStatus();
+        } catch (IOException e) {
+            logger.debug("Could not connect to OpenGarage controller", e);
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, e.getMessage());
+        } catch (RuntimeException e) {
+            logger.warn("Unexpected error connecting to OpenGarage controller", e);
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, e.getMessage());
+        }
+    }
+
+    private void pollStatus() throws IOException {
+        ControllerVariables controllerVariables = webTargets.getControllerVariables();
+        updateStatus(ThingStatus.ONLINE);
+        if (controllerVariables != null) {
+        logger.debug("Updating Thing/Items");
+            updateState(OpenGarageBindingConstants.CHANNEL_OG_DISTANCE, new QuantityType<>(controllerVariables.dist, MetricPrefix.CENTI(SIUnits.METRE)));
+            if (controllerVariables.door==0) {
+                logger.debug("Door state == closed, {}, {}",controllerVariables.door, false);
```

Comment:
why do you log a constant?

Reply:
I believe this is addressed in 4714fc9b7d. Stray code from when I was debugging why things weren't working as intended.